### PR TITLE
infix-to-postfix

### LINKED ISSUE #1772: Infix to Postfix in Python3
Please assign me this  and add the label GSSoC 20.

### PR DESCRIPTION
**Pull request by abhinashgiri**
Fixes #1772 
#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
-  A program to convert given infix expression to postfix expression and output the evaluated result.

#### Languages Used:
-  Python 3
#### Files Added:
- infix_to_postfix.py

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by abhinashgiri** [comment]
#1772

File: Infix_To_Postfix/Infix_to_Postfix.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    if (c == "+" or c == "-") and (b == "*" or b == "/"):
+        return True
+    elif b == "^":
+        return True
+
+# function to help evalute expression by performing different operations
+def perform(c, b, a):
+    d = 0
+    if b == "+":
+        d = int(a)+int(c)
```
Comment (SakshiShreya) [inline Infix_To_Postfix/Infix_to_Postfix.py]: Add space around operators.

File: Infix_To_Postfix/Infix_to_Postfix.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        12 4 + 5 * 8 9 * 5 2 ^ - -
+        33
+
+
+
+
+
+
+
+
```
Comment (SakshiShreya) [inline Infix_To_Postfix/Infix_to_Postfix.py]: Remove the extra new lines at the EOF. Only one new line  should be present.

File: Infix_To_Postfix/Infix_to_Postfix.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+#Sample Test Cases
+Input : ( 12 + 45 ) * 5 ^ 2
+Output: The Postfix of the given expression is 
+        12 45 + 5 2 ^ *
+        1425
+Input : ( 12 + 4 ) * 5 - 8 * 9 - 5 ^ 2
+Output: The Postfix of the given expression is 
+        12 4 + 5 * 8 9 * 5 2 ^ - -
+        33
```
Comment (SakshiShreya) [inline Infix_To_Postfix/Infix_to_Postfix.py]: Put these in comment.

**Comment by SakshiShreya** [comment]
Please squash your commits and complete the PR template

**Comment by SakshiShreya** [comment]
You have not worked on any of the points that I have mentioned in the review. Please work on them, and then squash the commits.

**Pull request merged**